source/documentation/3.9/rspec-expectations/RSpec.html

```diff
@@ -86,8 +87,8 @@
 
 <h2>Overview</h2><div class="docstring">
   <div class="discussion">
-    <p>RSpec&#39;s top level namespace. All of rspec-expectations is contained
-in the <code>RSpec::Expectations</code> and <code>RSpec::Matchers</code> namespaces.</p>
+    <p>TODO: Refactor this file to be under our class length
+rubocop:disable ClassLength</p>
```

Comment:
This needs to be excluded somehow....

Reply:
https://github.com/rspec/rspec-expectations/pull/1178